sdk/cdn/Azure.ResourceManager.Cdn/src/Generated/Models/SecretProperties.cs

```diff
@@ -18,14 +18,14 @@ public SecretProperties()
         /// <summary> object which contains secret parameters. </summary>
         internal SecretParameters Parameters { get; set; }
         /// <summary> The type of the Secret to create. </summary>
-        internal SecretType ParametersType
```

Comment:
Here `Type rename` and `Single property hide` work together, which make the name weird. But considering this is internal, so I'm ok with that.